.github/workflows/dev.yml

```diff
@@ -54,4 +54,5 @@ jobs:
             '!datafusion/CHANGELOG.md' \
             README.md \
             CONTRIBUTING.md
+          git config --global --add safe.directory "$GITHUB_WORKSPACE" 
```

Comment:
I think this is no longer needed after updating to latest master

Reply:
Reverted CI related change.